core/Cargo.toml

```diff
@@ -16,7 +16,9 @@ path = "lib.rs"
 anyhow = "1.0.57"
 deno_ops = { path = "../ops", version = "0.20.0" }
 futures = "0.3.21"
-indexmap = "1.8.1"
+# Stay on 1.6 to avoid a dependency cycle in ahash https://github.com/tkaitchuck/aHash/issues/95
+# Projects not depending on ahash are unafected as cargo will pull any 1.X that is >= 1.6.
+indexmap = "1.6"
```

Comment:
Is this still required?

Reply:
I believe so, there hasn't been much movement trying to break this dependency cycle https://github.com/tkaitchuck/aHash/issues/95. Luckily it doesn't pose any significant problems for Deno.

Reply:
Right, but we were already on `v1.8.1`, so why downgrade?

Reply:
This is not a real downgrade but a downgrade of the "minimum" minor version for `1.*`. `deno` `cargo.lock` still defines 1.8.1 and will go up as needed as long as it's `< 2`.

At the same time, if we don't downgrade the minimum minor version any crate depending on `deno_core` (like the project I'm trying to use deno in) also also `ahash` (or any of the other problematic dependencies) it won't compile as cargo won't be able solve the dependency cycle.